packages/lib/modules/lbp/steps/preview/ProjectedPrice.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  Divider,
+  Heading,
+  HStack,
+  Spacer,
+  Text,
+} from '@chakra-ui/react'
+import { ProjectedPriceChart } from '../sale-structure/ProjectedPriceChart'
+import { fNum } from '@repo/lib/shared/utils/numbers'
+import { useState } from 'react'
+import { differenceInDays, differenceInHours, parseISO } from 'date-fns'
+
+type Props = {
+  startTime: string
+  endTime: string
+  startWeight: number
+  endWeight: number
+  launchTokenSeed: number
```

Comment:
just pass as string as the number conversion happens below?

Reply:
It is more the other way around, started as string. Right now the conversion is not needed.